.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Ruby
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
let's change the main branch to `main` while we're here

Reply:
I'll go ahead and update it in `main.yml`, but I don't think I can update the default GitHub branch without admin privileges.  Would you be able to do that for me?